co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureRegistrationMessage.java

```diff
@@ -13,169 +13,161 @@
         return this.rxMessageIpAddress;
     }
 
     /**
-     * Returns the unique identifier for the Infrastructure Device
+     * Returns the unique identifier for the Infrastructure instance
      * 
-     * @return The unique identifier for the Infrastructure Device
+     * @return The unique identifier for the Infrastructure instance
      */
     public String getInfrastructureId() {
         return this.infrastructureId;
     }
 
     /**
-     * Returns the port number where the Infrastructure Device will be listening for
+     * Returns the port number where the Infrastructure instance will be listening for
      * inbound messages
      * 
-     * @return The port number where the Infrastructure Device will be listening for
+     * @return The port number where the Infrastructure instance will be listening for
      *         inbound messages
      */
     public int getRxMessagePort() {
         return this.rxMessagePort;
     }
 
     /**
-     * Returns the port number where the Infrastructure Device will be listening for
+     * Returns the port number where the Infrastructure instance will be listening for
      * time synchronization messages
      * 
-     * @return The port number where the Infrastructure Device will be listening for
+     * @return The port number where the Infrastructure instance will be listening for
      *         time synchronization messages
      */
     public int getTimeSyncPort() {
         return this.timeSyncPort;
     }
 
     /**
-     * Returns the Geo-coordinate of the Infrastructure Device location
+     * Returns the Geo-coordinate of the Infrastructure instance location
      * 
-     * @return The Geo-coordinate of the Infrastructure Device location
+     * @return The Geo-coordinate of the Infrastructure instance location
      */
     public CartesianPoint getLocation() {
         return this.location;
     }
 
     /**
-     * Sets the IP address where the Infrastructure Device will be listening for
-     * inbound messages
+     * Returns list of associated sensors/detectors.
      * 
-     * @param rxMessageIpAddress The IP address where the Infrastructure Device will
-     *                           be listening for inbound messages
+     * @return list of sensors/detectors registered to infrastructure instances.
      */
-    public void setRxMessageIpAddress(String rxMessageIpAddress) {
-        this.rxMessageIpAddress = rxMessageIpAddress;
+    public List<Detector> getSensors() {
+        return sensors;
     }
-
+    
     /**
-     * Sets the unique identifier for the Infrastructure Device
-     * It currently is the same with RSU ID
+     * Returns port where infrastructure instance will be listening for interaction messages
+     * from CDASim.
      * 
-     * @param infrastructureId The unique identifier for the Infrastructure Device
+     * @return int simulated interaction port
      */
-    public void setInfrastructureId(String infrastructureId) {
```

Comment:
Removed setters since they were unused and just require additional unit test code coverage.